src/business-logic/services/device-service-implementation.ts

```diff
@@ -22,4 +21,9 @@ export class DeviceServiceImplementation implements DeviceService {
     await this.deviceRepository.save(device)
     this.deviceEventEmitter.emitDeviceCreatedEvent(device)
   }
-}
+
+  public async update(device: Device): Promise<void> {
+    await this.deviceRepository.update(device)
+    this.deviceEventEmitter.emitDeviceUpdatedEvent(device) // Change to updated
```

Comment:
I'm a little confused about the the comment. It looks like that it might be a note for you to do later?
In any case, this comment doesn't provide much value for a maintenance developer, so it would be nice if it was deleted :)

Reply:
Yes, it was a note to self that got implemented then I forgot to update.
Nice catch.